shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -116,13 +116,29 @@
               .desc("reverse order while sorting")
               .build();
 
+  private static final Option USE_ACCESS_TIME_OPTION =
```

Comment:
instead of having 2 options for all these timestamps, we should have a single option like "--timestamp-format", which takes a string, and can be `m`, `c`, or `a`. 
- `m`: use modified time
- `c`: use creation time
- `a`: use access time

And the default value should be `m`, or whatever the actual default is. Would this work?

Reply:
How about `--timestamp <modified | created | accessed>`?
in general I'm prefer using longer and meaningful option names rather than a shorter but meaningless one-char options

Reply:
yeah that works. The main point is to have a single option that "picks" which timestamp to use for the time column.

Reply:
I agree we should use a single option. I am not sure how to implement that with apache common CLI APIs.

Regarding the option name, both HDFS and Unix use `-u` for listing access time and Unix use `-U` for listing creation time. Shall we keep the same convention to avoid mental overhead for user to remember different parameter scheme?

Reply:
I don't think `-u` is commonly known any way. Plus IMO, HDFS and Unix are not good examples to follow w.r.t. CLI syntax. Anyway, my point is we don't have to follow their approach for detailed option names.

Reply:
`-u` and `-U` seems confusing. For example, what happens if you use both at the same time? It seems like having a single option which controls the timestamp type is more intuitive.